### DISCUSSION TITLE
Improve 'action needed' attention in Dev tools Statistics tab

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#12317] [Dashboard]
As it stands, we have a huge scroll list on the statistics page, and in my case, not a single issue to be fixed. All I see are endless entities with either

No Issue
This entity is being excluded from being recorded
This entity is no longer being recorded.

All of these are intentional.

it would really be helpful is these different type of 'issues' could be somehow Folded under their respective headers with that description. It would allow the true offenders to stand out a first glance. Want to have a look? open fold, ready? close fold.

I am aware we can click the column header to group, but that still leaves us with a page of several hundreds (in my carefully curated list, possibly many more for users that dont set anything) of entities that dont need attention at all.

**Comment by rccoleman** [comment]
Agreed on the sentiment that there are a bunch of intentionally missing entities in `recorder:` that lead to persistent errors in DevTools->Statistics that obscure "real" issues.  Ideally, the "fix" would be similar to what happens when the units change - something like "Delete old statistics?".  It's especially annoying because users don't explicitly choose what gets included in long-term statistics; they're pulled in automatically based on characteristics of the entity.  So I didn't ask for them to be recorded, I stopped recording the history, and now I'm stuck with old, unwanted data and the accompanying error message unless I manually dig through the database to remove them.

Here's an example from a different issue that has the "Remove" option that would also work well for entities that have been excluded from recorder:
<img width="535" alt="image" src="https://user-images.githubusercontent.com/860888/181341629-1054ea0c-bdf2-4d7e-923c-777c0aff2ad2.png">

Being able to select multiple items and act on them all at once would also be a welcome addition for this type of issue.

**Comment by Mariusthvdb** [comment]
yes, and this is really just a frontend FR, but truly feels like an issue, as experienced in the new beta cycle. 
see: https://github.com/home-assistant/core/issues/75893, where some can be deleted and others cant. Its just no consistent at all.
